src/pages/Index.js

```diff
@@ -113,7 +113,7 @@ export default () => {
             <OurMissions />
             <Industry />
             <RoadMap />
-            <Partners />
+            {/* <Partners /> */}
```

Comment:
Partners component should not be commented out.  Have a look at main branch.  Partners section has been published already.